packages/transaction-controller/src/TransactionController.ts

```diff
@@ -2435,23 +2440,11 @@ export class TransactionController extends BaseControllerV1<
    * specified in txParams, @ethereumjs/tx is able to determine which EIP-2718
    * transaction type to use.
    *
+   * @param chainId
    * @returns common configuration object
    */
-  private getCommonConfiguration(): Common {
-    const {
-      providerConfig: { type: chain, chainId, nickname: name },
-    } = this.getNetworkState();
-
-    if (
-      chain !== RPC &&
-      chain !== NetworkType['linea-goerli'] &&
-      chain !== NetworkType['linea-mainnet']
-    ) {
-      return new Common({ chain, hardfork: HARDFORK });
-    }
-
+  private getCommonConfiguration(chainId: Hex): Common {
     const customChainParams: Partial<ChainConfig> = {
-      name,
```

Comment:
how was this used before 🤔 ?

Reply:
I don't believe it is really used

Links below are to a more recent version than the one we use, but still you can see that name isn't really used.

https://github.com/ethereumjs/ethereumjs-monorepo/blob/44e069e41b13eddd8cd09ab62e75f4641eb234d0/packages/tx/src/transactionFactory.ts#L19

https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/common/src/common.ts#L1084